samza-core/src/main/scala/org/apache/samza/storage/KafkaTransactionalStateTaskStorageBackupManager.scala

```diff
@@ -39,24 +39,28 @@ import scala.collection.JavaConverters._
 /**
  * Manage all the storage engines for a given task
  */
-class TransactionalStateTaskStorageManager(
+class KafkaTransactionalStateTaskStorageBackupManager(
```

Comment:
Do you foresee any shared logic in the commit path regardless of the choice of durability? Wonder if we should keep this as is and extract the remote store specifics out to the backup manager.

Reply:
We don't foresee any shared logic for now, we will change this in the future if we notice any common paths with the new remote store implementation